packages/website/src/lib/load-readme.ts

```diff
@@ -1,9 +1,6 @@
-import { existsSync, readFileSync } from 'node:fs';
+import { readFileSync } from 'node:fs';
 
 export const loadReadme = (): string => {
-  const fileName = existsSync('./README.md')
-    ? './README.md'
-    : '../../README.md';
-  const file = readFileSync(fileName, 'utf8');
+  const file = readFileSync('./private/README.md', 'utf8');
```

Comment:
We don't know if `./private/...` path in the code is confusing or not.
If it's not quite acceptable by users, let's consider removing the capability.